### ISSUE TITLE
Font looks blurry when scaled

### ISSUE THREAD

**Issue by gbrlsnchs** [kolunmi/sandbar#8]
I'm using a scaled resolution (1.5x) and, although the bar does scale along, the font rendering seems really blurry compared to other software I use (foot and dunst).

**Comment by kolunmi** [comment]
Hey! sorry for the delay. I introduced the `-scale` option in my most recent commit c731bec. Does this fix your issue?

**Comment by itscool900** [comment]
> Hey! sorry for the delay. I introduced the `-scale` option in my most recent commit [c731bec](https://github.com/kolunmi/sandbar/commit/c731bec75f3cdfac5d257494ea355e4c1519d7d3). Does this fix your issue?

No, this only supports integer scaling. You'd need to support fractional scaling, too.

**Comment by itscool900** [comment]
Also, I tried using it with an integer, and I just got `Option '' not recognized`

**Comment by itscool900** [comment]
Experiencing the same problem--the new feature works fine on my end. However, it's worth documenting that -scale must be placed at the end of the list of options or it fails with `Option '' not recognized`.
